### PR TITLE
ROE-2405 Trusts associated with OE

### PR DESCRIPTION
**Pull request by Zain-Abbas1**
### JIRA link
https://companieshouse.atlassian.net/browse/ROE-2405

### Change description
- Update index files to include new url with params
- Update html for change link with params
- Update trust involved page to redirect to next page with url params
- Update add.trust to include ActiveSubmissionBasePath in html template

### Work checklist

- [x] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: test/controllers/add.trust.controller.spec.ts [hunk 1]
```diff
@@ -91,6 +103,46 @@ describe("Add Trust Controller Tests", () => {
     });
   });
 
+  describe("GET tests with url params", () => {
+    test(`renders the ${config.ADD_TRUST_PAGE} page`, () => {
+      mockGetUrlWithParamsToPath.mockReturnValueOnce(MOCKED_URL);
+      (getApplicationData as jest.Mock).mockReturnValue(mockAppData);
+
+      mockIsActiveFeature.mockReturnValueOnce(true);
```
Comment (dlloyd1-ch) [inline test/controllers/add.trust.controller.spec.ts:111]: Worth adding a comment to document which flag we are mocking.

File: test/controllers/add.trust.controller.spec.ts [hunk 2]
```diff
@@ -91,6 +103,46 @@ describe("Add Trust Controller Tests", () => {
...
+    });
+
+    test('catch error when renders the page with url params', () => {
+      const error = new Error(ANY_MESSAGE_ERROR);
+
+      (mockRes.render as jest.Mock).mockImplementationOnce(() => {
+        throw error;
+      });
+
+      mockIsActiveFeature.mockReturnValueOnce(true);
```
Comment (dlloyd1-ch) [inline test/controllers/add.trust.controller.spec.ts:137]: Add comment as above.

**Comment by markpit** [comment]
looks ok to me once dans's comments resolved

File: test/controllers/add.trust.controller.spec.ts (hunk 1 above)
Comment (Zain-Abbas1) [reply test/controllers/add.trust.controller.spec.ts]: Done

File: test/controllers/add.trust.controller.spec.ts (hunk 2 above)
Comment (Zain-Abbas1) [reply test/controllers/add.trust.controller.spec.ts]: Done

**Review by dlloyd1-ch — APPROVED** [review]

**Pull request merged**